tests/ui/rust-2024/unsafe-env.rs

```diff
@@ -0,0 +1,30 @@
+//@ revisions: e2021 e2024
+//@[e2021] edition: 2021
+//@[e2024] edition: 2024
+//@[e2024] compile-flags: -Zunstable-options
+
+use std::env;
+
+unsafe fn unsafe_fn() {}
+fn safe_fn() {}
+
+#[deny(unused_unsafe)]
+fn main() {
+    env::set_var("FOO", "BAR");
+    //[e2024]~^ ERROR [E0133]
```

Comment:
```suggestion
    //[e2024]~^ ERROR call to unsafe function `set_var` is unsafe
```
Could you use human readable error annotations in all tests?

Reply:
All these annotations are for humans to double check, if something changes a human should go and manually verify and update the new error text.

Technically we could not have these annotations at all because all the errors are already in `.stderr` files, but that would just gradually turned the test suite into garbage because it's hard to match the contents of `.rs` and `.stderr` files during review (especially given that stderr files don't have line numbers now). In testing redundancy is good.

Reply:
Fixed. Thanks for the explanation!